packages/backend/src/api/middlewares/lang.ts

```diff
@@ -0,0 +1,13 @@
+import type { NextFunction, Request, Response } from 'express';
+import { DEFAULT_LOCALE } from '@config';
+import { langService } from '../../lang';
+
+export const langMiddleware = (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  const locale = req.cookies?.locale_name ?? DEFAULT_LOCALE;
```

Comment:
We will send current language in headers. Contact Ivan

Reply:
Changed cookies to headers, added the name of locale header to shared, so Ivan can change it without problems for us.